nested/network.json

```diff
@@ -185,7 +185,7 @@
             "sku": {
                 "name": "[parameters('lampCommon').lbSku]"
             },
-            "apiVersion": "2017-10-01",
+            "apiVersion": "latest",
```

Comment:
Let's not use "latest" for apiVersion, let's have a unique date string around this commit/time-frame for every applicable mention of apiVersion.